chain/jsonrpc/Cargo.toml

```diff
@@ -5,17 +5,17 @@ authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
 [dependencies]
-actix = "0.9"
-actix-web = "2"
-actix-cors = "0.2"
-tokio = { version = "0.2", features = ["full"] }
+actix = "0.11.0-beta.1"
+actix-web = "4.0.0-beta.1"
+actix-cors = { git = "https://github.com/andy128k/actix-extras.git", branch="update-dependencies" }
```

Comment:
Depending on a random branch seems brittle to me. What happens if the user deletes the branch? Further down you list the dependency as `0.5.4`; is there something in particular we need here that is not needed there? If so, maybe we should make our own fork of `actix-extras` where we can keep this branch around until we are able to upgrade to the next release of `actix-cors`.

Reply:
I need beta versions of actix with tokio 1.x support, and actix-cors has not been released with the version bump yet. Forking to our guthub org makes perfect sense, I will do that 👍